djstripe/migrations/0018_alter_invoiceitem_amount.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.13 on 2022-05-27 17:15
+
+from django.db import migrations
+
+import djstripe.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("djstripe", "0017_auto_20220524_1836"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="invoiceitem",
+            name="amount",
+            field=djstripe.fields.StripeQuantumCurrencyAmountField(),
```

Comment:
This migration will most likely not work as you intend it to. The amounts will be wrong. Try it with real data and report back please.